app_dart/lib/src/datastore/cocoon_config.dart

```diff
@@ -362,7 +362,7 @@ class Config {
     final Map<String, dynamic> appInstallations = await githubAppInstallations;
     final String appInstallation =
         appInstallations['$owner/$repository']['installation_id'] as String;
-    final String jwt = await generateJKT();
+    final String jwt = await generateJsonWebToken();
```

Comment:
Nit: all jwt -> jsonWebToken?

Reply:
The previous line was deleted and the remaining ones are coming from a library. Is there a reference that I missed?

Reply:
Oh, I meant all corresponding variables. `final String jwt` => `final String jsonWebToken`.